tests/e2e/scenarios/test_reborn_webui_v2_legacy_approval.py

```
@@ -0,0 +1,541 @@
+            "headline": "Approval required",
+            "body": "Allow shell to inspect the workspace?",
+            "allow_always": allow_always,
+            "approval_context": {
+                "tool_name": "builtin.shell",
+                "reason": "Allow shell to inspect the workspace?",
+                "action": {"label": "Run command", "preview": long_command},
+                "destination": {"label": "Local workspace"},
+                "scope": {"label": "Workspace"},
+                "details": [
+                    {"label": "Command", "value": long_command},
+                    {"label": "Directory", "value": "/tmp/reborn-approval"},
```

Ruff (0.15.20)

[error] Probable insecure usage of temporary file or directory: "/tmp/reborn-approval"

(S108)